jest.config.js

```diff
@@ -230,6 +233,12 @@ const jestConfig = {
         })),
         configureProject('pagebuilder', 'Pagebuilder', testReactComponents),
         configureProject('peregrine', 'Peregrine', inPackage => ({
+            // Make sure we can test extension files.
+            moduleFileExtensions: [
```

Comment:
I was suggesting to use `['ee.js', 'ce.js', 'js', 'json', 'jsx', 'node']` as the value of `moduleFileExtensions`.

Reply:
And you'd prefer that over `['ee.js', 'ce.js', ...defaults.moduleFileExtensions]` so we're explicit?

Reply:
Probably easiest just to change this to Revanth's suggestion, @larsroettig

Reply:
> And you'd prefer that over `['ee.js', 'ce.js', ...defaults.moduleFileExtensions]` so we're explicit?

I like the idea of maintaining these in a global location somewhere and importing them in the jest and webpack config files instead of relying on `jest.moduleFileExtensions` which can change in a future version.

I don't think we can use 1 array for both since the syntaxes they employ are different.

For instance,

Webpack config: `['.ee.js', '.ce.js', '.js', '.json', '.jsx', '.node']`
Jest config: `['ee.js', 'ce.js', 'js', 'json', 'jsx', 'node']`

Webpack needs `.` in front of the file extensions while jest doesn't.

The best solution at the moment would be to revert this change to what it was in `develop` and handle it in a different ticket if deemed necessary.